docs/content/users/usage/commands.md

```diff
@@ -1032,7 +1032,7 @@ ddev sequelace
 
 ## `sequelpro`
 
-Open [SequelPro](https://www.sequelpro.com/) with the current project’s database (global shell host container command). This command is only available if `Sequel Pro.app` is installed as `/Applications/Sequel pro.app`, and only for projects with `mysql` or `mariadb` databases.  *Note that Sequel Pro is considered abandoned, does not work with `mysql:8.0`; other solutions like Sequel Ace, Querious, Tableplus are preferred.
+Open SequelPro with the current project’s database (global shell host container command). This command is only available if `Sequel Pro.app` is installed as `/Applications/Sequel pro.app`, and only for projects with `mysql` or `mariadb` databases.  *Note that Sequel Pro is abandoned, does not work with `mysql:8.0`; other solutions like Sequel Ace, Querious, Tableplus are preferred.
```

Comment:
“Sequel Pro” should be two words, “TablePlus” should be upper-T-P, this is worth being a little louder about for people relying on an abandoned tool, and “are preferred” is impersonal language [to avoid](https://ddev.readthedocs.io/en/stable/developers/writing-style-guide/#use-active-third-person).

```suggestion
!!!warning "Sequel Pro is abandoned!"
    The project is abandoned and doesn’t work with MySQL 8. We recommend Sequel Ace, Querious, and TablePlus.
    
Open Sequel Pro with the current project’s database (global shell host container command). This command is only available if `Sequel Pro.app` is installed as `/Applications/Sequel pro.app`, and only for projects with `mysql` or `mariadb` databases.
```

Reply:
Checked the formatting to be sure the callout looks like we’d hope:

<img width="600" alt="Screen Shot 2023-03-20 at 05 08 14 PM@2x" src="https://user-images.githubusercontent.com/2488775/226491554-78017b13-1275-44b4-b41c-5c5119623708.png">

Reply:
Shucks, sorry I moved along so fast! Was trying to fix all the builds that were broken by sequelpro.com being broken, perhaps forever.